packages/core/util/util.js

```diff
@@ -110,6 +110,28 @@ const extractAriaProps = function (props) {
 	return aria;
 };
 
+/**
+ * Removes voice control related props from `props` and returns them in a new object.
+ *
+ * @function
+ * @param   {Object}    props    Props object
+ *
+ * @returns {Object}             voice control related props
+ * @memberof core/util
+ * @public
+ */
+const extractVoiceProps = function (props) {
```

Comment:
I'd like to move this method into moonstone and mark it private since voice control is currently platform-specific and `core` should be platform-independent.

Reply:
@ryan I also agree your guide and move the function to moonstone. (I did made moonstone/internal/util.js because I can't find proper file to move it. If you think the function should be placed another js file in moonstone, please mention it and I will change it)